src/response_logic.py

```diff
@@ -39,15 +43,30 @@ async def create_response(text: str, name: str, chat_id: int) -> str:
                 return f"Your account was already inactive."
 
         elif first_word == "send" or first_word == "/send":
-            memory = select_random_memory(user)
-            if memory is None:
-                return (
-                    "The user is not in the system, please join by typing; *join* or /join"
-                )
-            elif memory is False:
-                return "No memory found in the system. Please add memory, for further information you can type *help* or /help"
+            if len(split_text) == 1:
+                memory = select_random_memory(user)
+                if memory is None:
+                    return (
+                        "The user is not in the system, please join by typing; *join* or /join"
+                    )
+                elif memory is False:
+                    return "No memory found in the system. Please add memory, for further information you can type *help* or /help"
+                else:
+                    return memory.reminder
             else:
-                return memory.reminder
+                try:
+                    number_of_sending = int(split_text[1])
+                    if not (1 <= number_of_sending < 50):
+                        return "Please give a number which is 1<n<50  after *send*"
```

Comment:
This value can be set to 1. So this text should change to "0<n<50"

Reply:
well here is the story:

"send" sends 1 memory,
"send n" send n memory, I thought users should not use "send 1" but did not want to prohibit it :D